typings/embedded_io/socket/tcp/tls.d.ts

```diff
@@ -21,12 +21,5 @@ declare module "embedded:io/socket/tcp/tls" {
 	};
 	export type TLSDevice = TCPOptions & { io: typeof UDP };
 
-	export default class TLSSocket {
-		constructor(options: TLSOptions);
-		close(): undefined;
-		read(count: number | ArrayBufferLike): undefined | ArrayBufferLike;
-		write(buffer: ArrayBufferLike): number;
-		set format(format: string);
-		get format(): string;
-	}
+	export default class TLSSocket extends TCP {}
```

Comment:
I'm no TypeScript expert, but don't you need to define that the TLSSocket constructor uses `TLSOptions` instead of `TCPOptions` for the options object argument?